opentracing-shim/src/opentracingshim/__init__.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+
+import opentracing
+
+from opentelemetry.trace import Event
+from opentracingshim import util
+
+
+def create_tracer(tracer, scope_manager=None):
```

Comment:
```suggestion
def create_tracer(otel_tracer, scope_manager=None):
```

Alternatively, add type annotations (they also work fine as documentation if we don't run mypy).

Reply:
I'm working on adding both type info and proper documentation so this should be clear. I can still rename the argument *in addition* if you think it makes things clearer.

Reply:
One of renaming or type annotation is enough, I think (but I won't object to doing both either).

Reply:
Renamed. This should anyway become clearer when I'm done with the documentation (in a separate PR).